src/Extend/Middleware.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extend;
+
+use Flarum\Extension\Extension;
+use Illuminate\Contracts\Container\Container;
+use Zend\Stratigility\MiddlewarePipe;
+
+class Middleware implements ExtenderInterface
+{
+    private $middlewares = [];
+    private $stack;
+
+    public function __construct(string $stack)
```

Comment:
```suggestion
    public function __construct(string $frontend)
```

Let's make the naming of the Frontend we're impacting uniform. Other implementation like the one to add css/js etc uses frontend for this variable. Please modify here and the property.

Reply:
Will do, I was unsure on the name of that one since the original `ConfigureMiddleware` used `$stackName`